service/src/main/java/bio/terra/workspace/app/configuration/external/FeatureConfiguration.java

```diff
@@ -41,6 +42,14 @@ public void setTpsEnabled(boolean tpsEnabled) {
     this.tpsEnabled = tpsEnabled;
   }
 
+  public boolean isBpmEnabled() {
```

Comment:
I have flagged out the use of BPM so we are backwards compatible in environments where BPM is not deployed.